### PR TITLE
Add message listener

### PR DESCRIPTION
**Pull request by aleksey-slavik**
1) add message listener
2) add message listener test

### CODE REVIEW

File: src/main/java/com/globallogic/listener/MessageListener.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@Component
+@EnableSqs
+public class MessageListener {
+
+    /**
+     * Listener for queue with name 'spring-boot-queue'
+     *
+     * @param message received message
+     */
+    @SqsListener("spring-boot-queue")
```
Comment (HMRoot) [inline src/main/java/com/globallogic/listener/MessageListener.java]: [ISSUE] Lets move queue name to the application.properties. And let's change properties file to .yaml

File: src/test/java/com/globallogic/integration/MessageListenerTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Message listener test
+ *
+ * @author oleksii.slavik
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MessageListenerTest {
```
Comment (HMRoot) [inline src/test/java/com/globallogic/integration/MessageListenerTest.java]: [COSMETIC] Woul be better to have Integration word in this test's name. MessageListenerIntegrationTest. Also lets add unit test with the name MessageListenerTest.

File: src/main/java/com/globallogic/listener/MessageListener.java (hunk 1 above)
Comment (HMRoot) [reply src/main/java/com/globallogic/listener/MessageListener.java]: [COSMETIC] Please change queue name to e.g. "test-queue" or something. Its a bit confusing the same name for queue and application.

Comment (aleksey-slavik) [reply src/main/java/com/globallogic/listener/MessageListener.java]: [DONE] Migrate queue config data to application.yml file. Queue renamed to "test-queue"

File: src/test/java/com/globallogic/integration/MessageListenerTest.java (hunk 2 above)
Comment (aleksey-slavik) [reply src/test/java/com/globallogic/integration/MessageListenerTest.java]: [DONE] Renamed integration tests. Add unit test for queue listener

**Pull request merged**